cmd/karavictl/cmd/storage_delete.go

```diff
@@ -95,6 +95,8 @@ var deleteCmd = &cobra.Command{
 			log.Println("no storage of type", input.Type)
 			return
 		}
+		// TODO(aaron): investigation the quoting issue
+		// karavictl storage delete -t powermax -s '"000197901503"'
```

Comment:
Use a generic ID instead of an actual internal one

Reply:
This comment is outdated. I removed it.